src/components/Tab/index.ts

```diff
@@ -1,2 +1,2 @@
 export { Tab as default } from './Tab';
-export { Props as TabProps } from './Tab';
+export type { Props as TabProps } from './Tab';
```

Comment:
Dang, seems like `@typescript-eslint/consistent-type-imports` could've/should've found this 😞